packages/core/contracts-ovm/insured-bridge/implementation/AVM_BridgeDepositBox.sol

```diff
@@ -24,19 +24,17 @@ contract AVM_BridgeDepositBox is BridgeDepositBox, AVM_CrossDomainEnabled {
 
     /**
      * @notice Construct the Arbitrum Bridge Deposit Box
-     * @param _inbox Contract that sends generalized messages to the Arbitrum chain.
      * @param _l2GatewayRouter Address of the Arbitrum L2 token gateway router for sending tokens from L2->L1.
```

Comment:
this contract is within `contracts-ovm` but it's an AVM contract. 

for now, this is probs fine. once optimism is running sol 0.8 we can remove this directory in its totality.

Reply:
Yep, problem I explained [here](https://github.com/UMAprotocol/protocol/pull/3396#issuecomment-928171398) where we want `BridgeDepositBox` to use Solidity 0.8 since its re-used for the AVM/OVM DepositBoxes and I don't want to separate that contract from this one.